### PR TITLE
Fix CircleCI test failures from PRs #244 and #234

### PR DESCRIPTION
**Pull request by shortaflip**
CircleCI tests failed for the following:
Screen Shot 2021-03-18 at 11 20 40 AM

Please run `yarn test` prior to pull requests as this tests for potential lint errors that the project enforces.

### CODE REVIEW

File: modules/cmpapi/test/CallResponder.test.ts [hunk 1]
```diff
@@ -98,23 +98,27 @@ describe('CallResponder', (): void => {
...
     const callResponder = new CallResponder({
       [TCFCommand.GET_TC_DATA]: (): void => {
+
         customCommandCallback();
+
       },
     });
-  
+
+    CmpApiModel.tcModel = TCModelFactory.withGVL();
```
Comment (shortaflip) [inline modules/cmpapi/test/CallResponder.test.ts:110]: Looks like without a tcModel defined, the code path leads down this path:
Screen Shot 2021-03-18 at 11 24 38 AM

**Comment by shortaflip** [comment]
@renebaudisch and @kemalmustafic 

Hey guys, just tagging you to let you know that some code needed to be changed from your PRs to fix some test failures in CircleCI.

**Comment by alextcone** [comment]
> @renebaudisch and @kemalmustafic
> 
> Hey guys, just tagging you to let you know that some code needed to be changed from your PRs to fix some test failures in CircleCI.

@shortaflip - I think @renebaudisch is new to testing in an app like this so we may need to go to the broader group. I'm going to ping the slack for reviewers.

**Comment by shortaflip** [comment]
@alextcone Oh, aha yeah I wasn't tagging them for a code review.  I just wanted them to know their were changes to the original code that they submitted.  And thank you for the help.

**Comment by renebaudisch** [comment]
Yeah, and I'm not really sure if my changes cause these failures, I don't really think so. But if I have a bit of time I will still have a look :)

**Comment by shortaflip** [comment]
@renebaudisch hey, no worries.  The ones from your side were just some untyped variables that the lint tests from the CI caught.  It is already a part of the pull request, so you are good to go.

**Pull request merged**